src/lib.rs

```diff
@@ -37,18 +37,30 @@ The name of this library is inspired from the famous surrealist artist `Salvador
 
 extern crate nalgebra as na;
 extern crate num_traits as num;
-#[cfg(all(feature = "dim2", feature = "parry"))]
+#[cfg(all(feature = "dim2", feature = "f32"))]
 pub extern crate parry2d as parry;
-#[cfg(all(feature = "dim3", feature = "parry"))]
+#[cfg(all(feature = "dim2", feature = "f64"))]
+pub extern crate parry2d_f64 as parry;
+#[cfg(all(feature = "dim3", feature = "f32"))]
 pub extern crate parry3d as parry;
-#[cfg(all(feature = "dim2", feature = "rapier"))]
+#[cfg(all(feature = "dim3", feature = "f64"))]
```

Comment:
"rapier" feature will still be needed if we want to keep salva optionally independent from parry.